vnext/ReactUWP/Modules/NativeUIManager.cpp

```diff
@@ -1164,6 +1164,69 @@ void NativeUIManager::measure(
   callback(args);
 }
 
+void NativeUIManager::findSubviewIn(
+    facebook::react::ShadowNode &shadowNode,
+    float x,
+    float y,
+    facebook::xplat::module::CxxModule::Callback callback) {
+  ShadowNodeBase &node = static_cast<ShadowNodeBase &>(shadowNode);
+  auto view = node.GetView();
+
+  auto rootUIView = view.as<winrt::UIElement>();
+  if (rootUIView == nullptr) {
+    callback({});
+    return;
+  }
+
+  winrt::Point point{x, y};
+
+  // Transform the point to app window coordinates because
+  // FindElementsInHostCoordinate() work in that metric
+  auto appWindowTransform = rootUIView.TransformToVisual(nullptr);
```

Comment:
this code is pretty similar to TouchEventHandler::GetTagsAtPoint, should share or have a comment about keeping them in sync when changes are made ?

Reply:
OK, I'll share the code.

Reply:
I gave it a try, but the two functions have quite a bit of differences. My function only needs to return the first point, and it also needs the returned the hit test element. The existing function `GetTagsAtPoint()` does not return the elements, and it returns the whole set of points. 

In the end, I don't think it's worth the effort to try to share the code here. The only two lines of code that they have in common are the calls to TransformToVisual() and FindElementsInHostCoordinates().